### PR TITLE
Update UUID version to 8.3.2

### PR DESCRIPTION
**Pull request by darkmatter18**
When you install this module (as a dependency), the following message appears on the console:

```bash
npm WARN deprecated uuid@3.4.0: Please upgrade to version 7 or higher. Older versions may use Math.random() in certain circumstances, which is known to be problematic. See https://v8.dev/blog/math-random for details.
```

That's why I am making this PR.

### CODE REVIEW

**Comment by mroderick** [comment]
This is a duplicate of #20, it suffers the same failures on Linux and macOS.

Perhaps you and @SebastianBar can collaborate to figure out why this fails and push it across the finish line?

**Comment by darkmatter18** [comment]
@mroderick Why the CI isn't running?

File: index.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 'use strict'
-const uuid = require('uuid/v4')
+const { v4: uuid } = require('uuid');
```
Comment (SebastianBar) [inline index.js:2]: Just noticed they don't use `;` at the end. Please remove it for consistency 😄

Comment (darkmatter18) [reply index.js]: Done 😄

**Comment by darkmatter18** [comment]
@mroderick So as you can see, I tried a lot of hit and trial, and I come to a conclusion.

The error in the CI is not related to `UUID`, it is related to `node-tap`. It suffers failure in Travis `node_js:node` branch. 
The reason is previously when the master branch got a successful CI, the Node version in `v13.12.0`
Now when I am going through the same pipeline, I am getting an error, cause the Node Version is upgraded to `v16.4.0` and `TAP` version `^14.6.5` doesn't support Node 16.

> So here is the report, this repo has 2 outdated libraries `UUID` and `TAP`. 

Now to fix the `Tap` error, you have to upgrade to `Tap 15`, but again `tap 15` doesn't support `Node 8`.
That's it.

**Comment by hexeberlin** [comment]
@coreyfarrell Hi! @mroderick and I are interested in finishing this PR, can we help take it across the finish line?

**Comment by gboer** [comment]
Any reason why support for Node 8 can't be dropped to fix this? A lot of other packages have already dropped support for it and some don't even run anymore when using Node 8. So it shouldn't break a lot of builds anymore.

**Comment by natlibfi-jonollil** [comment]
Npm does not support node 8, 9, 11, 13 or 15 anymore.
Also npm 8 has been released and it dropped support for node 10, so to gain maximum support time should this go straight to 16?

https://nodejs.org/en/about/releases/
https://github.blog/changelog/2021-10-07-npm-cli-upgraded-to-version-8/

**Comment by mroderick** [comment]
@coreyfarrell are you the maintainer of this package? Can we get this PR across the finish line somehow?

**Comment by davesag** [comment]
This is the one remaining dependency I have on the old `uuid`. It'd be lovely to get this over the line, and then `nyc` get updated to use it.

**Comment by gboer** [comment]
@darkmatter18 , could you change the PR so that the UUID package is updated and then drop support for Node 8 I'd say. Otherwise you cannot make it work, as you said. And then hopefully, @coreyfarrell can come along, check it and merge it.

**Comment by darkmatter18** [comment]
@gboer Pushing a new PR

**Comment by darkmatter18** [comment]
@gboer Created a new PR: #21 
@coreyfarrell Please review

**Pull request closed without merging**